### ISSUE TITLE
Vanilla "Apply ControlNet (Advanced)" breaks after loading extension

### ISSUE THREAD

**Issue by JosefKuchar** [FizzleDorf/AIT#21]
"Apply ControlNet (Advanced)" node is broken when this extension is loaded - without using any custom nodes. "Apply ControlNet" node seems to work fine

Error below:
```
!!! Exception during processing !!!
Traceback (most recent call last):
  File "C:\Users\josef\projects\ComfyUI_windows_portable_nvidia_cu118_or_cpu\ComfyUI_windows_portable\ComfyUI\execution.py", line 151, in recursive_execute
    output_data, output_ui = get_output_data(obj, input_data_all)
  File "C:\Users\josef\projects\ComfyUI_windows_portable_nvidia_cu118_or_cpu\ComfyUI_windows_portable\ComfyUI\execution.py", line 81, in get_output_data
    return_values = map_node_over_list(obj, input_data_all, obj.FUNCTION, allow_interrupt=True)
  File "C:\Users\josef\projects\ComfyUI_windows_portable_nvidia_cu118_or_cpu\ComfyUI_windows_portable\ComfyUI\execution.py", line 74, in map_node_over_list
    results.append(getattr(obj, func)(**slice_dict(input_data_all, i)))
  File "C:\Users\josef\projects\ComfyUI_windows_portable_nvidia_cu118_or_cpu\ComfyUI_windows_portable\ComfyUI\nodes.py", line 657, in apply_controlnet
    c_net = control_net.copy().set_cond_hint(control_hint, strength, (1.0 - start_percent, 1.0 - end_percent))
TypeError: ControlNet.set_cond_hint() takes from 2 to 3 positional arguments but 4 were given
```

Minimal workflow example for reproducing
[advanced_minimal_example.zip](https://github.com/FizzleDorf/AIT/files/12317431/advanced_minimal_example.zip)

Works fine without AIT in `custom_nodes`

**Comment by hlky** [comment]
@JosefKuchar Test the fix at #23

**Comment by JosefKuchar** [comment]
@hlky Works for me, thanks!
